### DISCUSSION TITLE
__pycache__ files

### DISCUSSION THREAD

**Discussion by CJ-Rose** [smartcontractkit/full-blockchain-solidity-course-py#55] [Q&A] [answered]
This is what my project folder looks like at the end of Lesson 5. It's ugly and I hate it, but I'm doing some reading and I think I can deal with it. But I'm curious why I have generated all these files that did not appear in the video. 

<img width="1018" alt="Screen Shot 2021-09-18 at 8 36 33 PM" src="https://user-images.githubusercontent.com/79813968/133914710-8e0693f3-4c22-4272-bbd1-91188fe66957.png">

EDIT:
I see that the code in the course resources has `__pycache__ `listed in `.gitignore`. Why didn't they appear while the project was in progress like mine?

**Comment by AlbertoSinigaglia** [comment, marked as answer]
`gitignore` is just a file that Git uses to exclude files and folders from the project tracking... since it's something that you are generating on your local device, you don't want those to be shared (for example, the `.vscode` is not tracked since it's personal, and coworkers don't care about it)... at the end, everything that starts with `.` (and so it's a hidden file/folder) should be ignored

**Reply by PatrickAlphaC** [reply]
100% please ignore those auto-generated files, it's just something that happens with python, you can remove them our force to remove them, but for now, you'll do great just to pretend they aren't even there!
